shared/src/simulation/behaviors/manage-patient-transport-to-hospital.ts

```diff
@@ -456,7 +456,14 @@ export const managePatientTransportToHospitalBehavior: SimulationBehavior<Manage
                                 );
                             const expectedManagedPatients = Object.values(
                                 expectedPatientsPerRegion
-                            ).reduce(addResourceDescription);
+                            ).reduce(addResourceDescription, {
```

Comment:
Nice, the [note about `Array.prototype.reduce` throwing an error when no `initialValue` is specified and the array is empty](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce#exceptions) is missing from the [TS docstring](https://github.com/microsoft/TypeScript/blob/main/src/lib/es5.d.ts#L1465-L1471).